include/libtorrent/disk_buffer_pool.hpp

```diff
@@ -96,7 +96,7 @@ namespace libtorrent {
 		}
 		int num_to_evict(int num_needed = 0);
 
-		void set_settings(aux::session_settings const& sett);
```

Comment:
I think I might prefer renaming ``disk_buffer_pool::set_settings`` to something else instead. Say, ``disk_buffer_pool::set_cache_size()`` or something like that